src/lib/packet.h

```diff
@@ -400,6 +401,17 @@ typedef struct {
                 * with x being the secret) */
 } pgp_elgamal_pubkey_t;
 
+/** Structure to hold an ECC public key params.
+ *
+ * \see RFC 6637
+ */
+typedef struct {
+    uint8_t oid_len;
+    uint8_t *oid;
+    BIGNUM *point; /* octet string encoded as MPI */
```

Comment:
Different name might make sense but I don't think it's right to have x,y as distinct pair here - rfc4880bis sec 13.2 (and also RFC 6637) specify that ECC points are stored as a single MPI which contains both x and y coordinates so I think this structure is correct for both EdDSA and ECDSA/ECDH.